src/main/java/core/service/impl/WriterServiceImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.service.impl;
+
+import core.service.WriterService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class WriterServiceImpl implements WriterService {
+    private static final String REPORT_FILEPATH = "src/main/java/resources/report.csv";
+
+    @Override
+    public void writeTo(List<String> list) {
+        try {
+            Files.write(Path.of(REPORT_FILEPATH), list);
```

Comment:
Do we create a new file in the file system by hand or automatically in here?

Reply:
This occurs automatically.